src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -238,8 +238,11 @@ public void EditGameResult(Game game)
 
             ValidateGame(game, tournamentScheduleInfo);
 
+            var gameInDtoToCheckVideo = Get(game.Id);
+
             // Add autogeneration
-            if (tournamentScheduleInfo.Scheme == TournamentSchemeEnum.PlayOff)
+            if (tournamentScheduleInfo.Scheme == TournamentSchemeEnum.PlayOff &&
+                game.UrlToGameVideo == gameInDtoToCheckVideo.UrlToGameVideo)
```

Comment:
What is this for?

Reply:
to know if user edits only video or all game

Reply:
Well, question why this check is here and why it is checking only game video? if we add another property in the future - what are the chances that we forget to add it here?

Reply:
it is forbidden to edit playoff after gameresults set, but we need to change video, so I check if only video changed gameresults updates